aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -7,6 +7,12 @@
 #include <ATen/TensorIndexing.h>
 #include <ATen/cpu/vec/vec256/vec256.h>
 
+// #ifdef USE_CUDA
+// #include <ATen/native/transformers/cuda/sdp_utils.h>
```

Comment:
nit

Reply:
Going to need this but for some reason this is not working